widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.kt

```diff
@@ -109,8 +109,8 @@ class AppBarView @JvmOverloads constructor(
         binding.title.text = title
     }
 
-    fun showToolbar() {
-        isVisible = true
+    fun showToolbar(visibility: Boolean = true) {
```

Comment:
An idea. Maybe `setVisibility(visibility: Boolean = true)` can be a better name for this function. This name will not overlap with `showToolbar(title: String?)` in `ChatView` and will better reflect that it can also hide toolbar, not only show.

Reply:
fixed